src/ProjectTemplates/ComponentsWebAssembly.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Server/Program.cs

```diff
@@ -22,6 +22,5 @@ public static void Main(string[] args)
                 {
                     webBuilder.UseStartup<Startup>();
                 });
```

Comment:
Can we also have a comment above `options.ProviderOptions.Authority = ...` saying something like:

```cs
// Configure your authentication provider options here
// For more information, see https://aka.ms/blazor-standalone-auth
```

I registered that fwlink just now.

Reply:
Only in the `MissingAuthority` case of course

Reply:
Wrong program.cs :)